packages/types/src/appearance.ts

```diff
@@ -176,6 +178,8 @@ export type ElementsConfig = {
   socialButtonsBlockButtonArrow: WithOptions<OAuthProvider | Web3Provider, never, never>;
   socialButtonsProviderIcon: WithOptions<OAuthProvider | Web3Provider, LoadingState, never>;
 
+  enterpriseButtonsProviderIcon: WithOptions<string, LoadingState, never>;
```

Comment:
If we remove the `elementId` from this element, we should replace `string` with `never`.

Reply:
Changed `string` to `SamldpSlug`.